framework/src/modules/dpos_v2/module.ts

```diff
@@ -178,19 +213,31 @@ export class DPoSModule extends BaseModule {
 	// eslint-disable-next-line @typescript-eslint/require-await
 	public async init(args: ModuleInitArgs) {
 		const { moduleConfig } = args;
-		const config = objects.mergeDeep({}, defaultConfig, moduleConfig) as ModuleConfigJSON;
+		const defaultGovernanceTokenID = `${args.genesisConfig.chainID}${Buffer.alloc(4).toString(
```

Comment:
Would `Buffer.alloc(CHAIN_ID_LENGTH).toString('hex')` would be correct?

Reply:
it's not necessarily `00000000` for the chainID, so we need to concat with the config chainID